pkg/admission/initializers/initializer.go

```diff
@@ -44,6 +47,26 @@ func (i *kcpInformersInitializer) Initialize(plugin admission.Interface) {
 	}
 }
 
+// NewKubeInformersInitializer returns an admission plugin initializer that injects
+// kube shared informer factories into admission plugins.
+func NewKubeInformersInitializer(
```

Comment:
I assume the upstream one is not exposed?

Reply:
Does this need anything ... ?